test/test-helpers/params.ts

```diff
@@ -0,0 +1 @@
+export const testnetPort = 48545
```

Comment:
Could the test work without this file?

Reply:
Yes, we can move this constant to `start-ganache.test.ts`. I just thought that some other tests may need it in the future, so I put it in the shared module.